packages/cluster-ui/src/barCharts/barCharts.tsx

```diff
@@ -63,6 +63,15 @@ const retryBars = [
   ),
 ];
 
+let total: number;
```

Comment:
We should avoid making a global variable. Is there a way to pass `total` down to the bar chart as an argument?

Reply:
done